src/test-run/index.js

```diff
@@ -53,6 +53,21 @@ const MAX_RESPONSE_DELAY              = 2 * 60 * 1000;
 
 const ALL_DRIVER_TASKS_ADDED_TO_QUEUE_EVENT = 'all-driver-tasks-added-to-queue';
 
+function getMarionetteClient (connection) {
```

Comment:
I propose to move this function to the `\src\browser\provider\built-in\firefox\get-marionette-client.js` file because it is not related to the `TestRun` class.